devstack/plugin.sh

```diff
@@ -3,6 +3,7 @@
 function install_heat_infoblox {
     cd $HEAT_INFOBLOX_DIR
     sudo python setup.py install
+    sudo pip install -r requirements.txt
```

Comment:
That works because pacakges from requirements are loaded only on heat start up, which happens after install_heat_infoblox is done.
But agree, from logical point of view it is more clear to install dependencies first and then package itself, so I'll update ther order.